packages/__vital__/src/shadow/@bodiless/vital-navigation/SubMenu.ts

```diff
@@ -20,7 +20,7 @@ const Footer = asSubMenuToken({
   ...vitalSubMenuBase.Footer,
   Compose: {
     ...vitalSubMenuBase.Footer.Compose,
-    _: addProps({ 'data-shadowed-by': '__vital__:FooterSubMenu' }),
+    _: addProps({ 'data-shadowed-by-2': '__vital__:FooterSubMenu' }),
```

Comment:
The same <li> element was being shadowed by another token, so the data-shadowed-by prop was being set there.  Now we get both data-shadowed-by and data-shadowed-by-2.  Not sure what you want to do with this.